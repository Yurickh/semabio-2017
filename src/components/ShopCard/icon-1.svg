<?xml version="1.0" encoding="UTF-8"?>
<svg width="93px" height="60px" viewBox="0 0 93 60" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 45.2 (43514) - http://www.bohemiancoding.com/sketch -->
    <title>cycle-1</title>
    <desc>Created with Sketch.</desc>
    <defs></defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="BetterDesk" transform="translate(-180.000000, -3367.000000)" fill-rule="nonzero" fill="#000000">
            <g id="cycle-1" transform="translate(180.000000, 3367.000000)">
                <g id="Group">
                    <path d="M79.8828358,0.62962963 C76.3432836,1.88888889 71.936194,3.44444444 61.5257463,2 C53.3708955,0.851851852 51.8093284,0.888888889 48.7208955,0.962962963 C47.8533582,0.962962963 46.8817164,1 45.5630597,1 C42.6134328,1 40.8436567,1.37037037 38.5533582,1.81481481 C36.0895522,2.2962963 33.0011194,2.92592593 26.9630597,3.40740741 C15.6156716,4.33333333 6.31567164,1.55555556 6.21156716,1.51851852 L6.00335821,1.48148148 C2.15149254,0.962962963 0.763432836,1.92592593 0.347014925,2.37037037 L0.347014925,4.44444444 C0.27761194,4.51851852 0.347014925,4.55555556 0.347014925,4.55555556 C0.347014925,4.55555556 2.94962687,4.03703704 5.55223881,4.37037037 C6.5238806,4.66666667 13.2212687,6.51851852 22.1395522,6.51851852 C23.7705224,6.51851852 25.436194,6.48148148 27.0671642,6.33333333 C33.2440299,5.85185185 36.4365672,5.22222222 38.9697761,4.7037037 C41.1906716,4.25925926 42.7869403,3.92592593 45.4936567,3.92592593 C46.8123134,3.92592593 47.8533582,3.92592593 48.7208955,3.88888889 C51.7052239,3.81481481 53.1973881,3.77777778 61.1440299,4.88888889 C71.3462687,6.2962963 76.0309701,5.03703704 80.7156716,3.37037037 C83.7,2.2962963 90.7791045,4.03703704 92.930597,4.81481481 L92.930597,2.07407407 C92.0630597,1.77777778 84.2552239,-0.925925926 79.8828358,0.62962963 Z" id="Shape"></path>
                    <path d="M49.033209,8.96296296 L48.616791,8.81481481 L48.2003731,8.96296296 C47.9921642,9.03703704 43.4115672,10.8148148 38.7962687,14.8148148 C32.5847015,20.1851852 29.2880597,26.9259259 29.2880597,34.3703704 C29.2880597,48.4074074 37.9634328,59.8518519 48.6514925,59.8518519 C59.3048507,59.8518519 68.0149254,48.4074074 68.0149254,34.4074074 C68.0149254,15.2962963 49.2067164,9.03703704 49.033209,8.96296296 Z M48.6514925,56.962963 C39.455597,56.962963 31.9947761,46.8148148 31.9947761,34.3703704 C31.9947761,19.9259259 46.0488806,13.037037 48.686194,11.8518519 C51.3235075,12.8518519 65.308209,18.9259259 65.308209,34.3703704 C65.308209,46.8148148 57.8473881,56.962963 48.6514925,56.962963 Z" id="Shape"></path>
                    <path d="M39.5597015,21 L39.1432836,20.4444444 C38.9350746,20.5925926 34.1462687,24.6666667 33.2440299,36.037037 L33.9033582,36.1111111 C34.7708955,25.1111111 39.4902985,21.037037 39.5597015,21 Z" id="Shape"></path>
                    <path d="M40.3231343,23.1851852 C40.1843284,23.4074074 36.6794776,28.5555556 36.1242537,35.8518519 L36.7835821,35.9259259 C37.3041045,28.8148148 40.8089552,23.6666667 40.8783582,23.6296296 L40.3231343,23.1851852 Z" id="Shape"></path>
                    <path d="M39.316791,34.9259259 L40.0108209,34.9259259 C40.0108209,30.7037037 42.058209,28.0740741 42.0929104,28.0740741 L41.5723881,27.5925926 C41.4682836,27.7037037 39.316791,30.4444444 39.316791,34.9259259 Z" id="Shape"></path>
                </g>
            </g>
        </g>
    </g>
</svg>