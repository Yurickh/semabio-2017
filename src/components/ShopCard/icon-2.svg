<?xml version="1.0" encoding="UTF-8"?>
<svg width="87px" height="96px" viewBox="0 0 87 96" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 45.2 (43514) - http://www.bohemiancoding.com/sketch -->
    <title>cycle-2</title>
    <desc>Created with Sketch.</desc>
    <defs></defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="BetterDesk" transform="translate(-424.000000, -3328.000000)" fill-rule="nonzero" fill="#000000">
            <g id="cycle-2" transform="translate(424.000000, 3328.000000)">
                <g id="Group">
                    <path d="M74.6952191,40.43861 C71.8876494,41.4393822 68.5601594,42.6254826 61.8011952,42.180695 C61.8705179,41.8841699 61.9398406,41.5505792 61.974502,41.2540541 C64.22749,28.392278 55.7007968,16.1606178 55.3541833,15.6416988 L54.3836653,14.2702703 L53.3784861,15.5675676 C51.3334661,18.1992278 49.6697211,20.7567568 48.3179283,23.2030888 C38.8553785,7.07953668 18.9250996,1.03783784 18.0585657,0.778378378 L16.5334661,0.333590734 L16.4641434,2.0015444 C15.4243028,21.5722008 20.138247,34.5081081 26.2733068,42.9590734 C25.9266932,42.996139 25.5454183,43.0332046 25.1641434,43.0702703 C14.5577689,43.9598456 5.82310757,41.365251 5.75378486,41.3281853 L5.58047809,41.2911197 C2.01035857,40.8092664 0.693227092,41.6988417 0.311952191,42.1065637 L0.311952191,44.0339768 C0.242629482,44.1081081 0.311952191,44.1451737 0.311952191,44.1451737 C0.346613546,44.1451737 2.77290837,43.6633205 5.19920319,43.9598456 C6.13505976,44.219305 12.3741036,45.96139 20.72749,45.96139 C22.2525896,45.96139 23.8123506,45.9243243 25.3374502,45.7760618 C26.3772908,45.7019305 27.3478088,45.5907336 28.2143426,45.5166023 C31.0219124,48.8525097 34.0027888,51.372973 36.8103586,53.2633205 C36.6023904,53.411583 36.4290837,53.5969112 36.2557769,53.7081081 C30.4673307,58.7119691 27.3824701,65.0501931 27.3824701,72.0185328 C27.3824701,85.2138996 35.5278884,95.9258687 45.5103586,95.9258687 C55.4928287,95.9258687 63.638247,85.2138996 63.638247,72.0555985 C63.638247,63.5305019 59.6521912,57.7111969 55.4581673,53.9675676 C58.0577689,51.2617761 59.9294821,48.1853282 61.0733068,44.8123552 C68.0055777,45.3683398 71.7836653,44.2934363 75.4924303,42.9590734 C78.2653386,41.9953668 84.8856574,43.6262548 86.8960159,44.3305019 L86.8960159,41.772973 C86.0988048,41.5135135 78.785259,38.9930502 74.6952191,40.43861 Z M47.6940239,27.5397683 C51.2294821,35.2123552 51.5760956,43.9598456 48.6645418,53.5227799 C44.9904382,26.5760618 24.7135458,10.1559846 18.8557769,5.93050193 C18.8557769,5.26332046 18.8904382,4.596139 18.8904382,3.92895753 C24.0549801,5.74517375 40.8657371,12.8247104 47.6940239,27.5397683 Z M43.8812749,54.2640927 C38.7167331,31.1722008 23.847012,17.6432432 19.063745,13.788417 C18.959761,12.6393822 18.9250996,11.4532819 18.8904382,10.2301158 C23.3964143,13.8254826 41.7322709,29.8378378 45.6143426,54.9312741 C45.059761,54.7459459 44.5051793,54.5235521 43.8812749,54.2640927 Z M46.3075697,55.1907336 C42.3908367,28.8 22.6338645,12.3057915 18.8557769,9.37760618 C18.8211155,8.52509653 18.8211155,7.63552124 18.8211155,6.74594595 C25.0948207,11.3420849 45.0944223,27.984556 48.1792829,54.9312741 C48.1099602,55.1907336 48.0059761,55.4131274 47.9366534,55.6725869 C47.5207171,55.56139 46.9661355,55.4131274 46.3075697,55.1907336 Z M19.1330677,14.6779923 C24.2282869,18.8664093 38.0928287,31.9876448 43.1533865,53.9305019 C35.1119522,50.2980695 21.3513944,40.3644788 19.1330677,14.6779923 Z M61.0733068,72.0555985 C61.0733068,83.7312741 54.0370518,93.2200772 45.4756972,93.2200772 C36.9143426,93.2200772 29.912749,83.7312741 29.912749,72.0555985 C29.912749,63.7528958 34.8346614,58.1559846 39.2366534,54.7830116 C44.4011952,57.7482625 48.387251,58.5637066 48.5258964,58.5637066 L49.5310757,58.7490347 L49.5310757,58.7490347 L49.5310757,58.7490347 L50.2936255,58.2301158 C51.4374502,57.4517375 52.5119522,56.6362934 53.5171315,55.7837838 C57.3645418,59.1196911 61.0733068,64.3459459 61.0733068,72.0555985 Z M59.4788845,40.7351351 C58.5430279,45.9243243 55.7701195,50.3722008 51.1601594,54.0046332 C54.1756972,43.8857143 53.759761,34.619305 49.947012,26.3166023 C49.8776892,26.1683398 49.8083665,26.0571429 49.7737052,25.9459459 C50.9521912,23.6478764 52.4079681,21.23861 54.2450199,18.7181467 C56.3247012,22.2023166 61.1079681,31.4687259 59.4788845,40.7351351 Z" id="Shape"></path>
                    <path d="M38.3007968,58.6007722 L37.9195219,58.0818533 C37.7115538,58.2671815 32.6856574,62.5667954 31.7498008,74.5389961 L32.3737052,74.6131274 C33.2749004,62.9745174 38.2661355,58.6749035 38.3007968,58.6007722 Z" id="Shape"></path>
                    <path d="M38.5780876,61.2324324 C38.5087649,61.2694981 34.6960159,66.8664093 34.1414343,74.5760618 L34.7653386,74.6131274 C35.3199203,67.1258687 39.0286853,61.6772201 39.098008,61.6401544 L38.5780876,61.2324324 Z" id="Shape"></path>
                    <path d="M37.4342629,72.5745174 L38.0928287,72.5745174 C38.0928287,68.6455598 39.9992032,66.1621622 39.9992032,66.1250965 L39.5139442,65.6803089 C39.4446215,65.7915058 37.4342629,68.3490347 37.4342629,72.5745174 Z" id="Shape"></path>
                </g>
            </g>
        </g>
    </g>
</svg>