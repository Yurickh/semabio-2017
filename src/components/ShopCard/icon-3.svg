<?xml version="1.0" encoding="UTF-8"?>
<svg width="101px" height="102px" viewBox="0 0 101 102" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 45.2 (43514) - http://www.bohemiancoding.com/sketch -->
    <title>cycle-3</title>
    <desc>Created with Sketch.</desc>
    <defs></defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="BetterDesk" transform="translate(-658.000000, -3306.000000)" fill-rule="nonzero" fill="#000000">
            <g id="cycle-3" transform="translate(658.000000, 3306.000000)">
                <g id="Group">
                    <path d="M1.403861,91.6795276 C1.44285714,91.6795276 2.33976834,91.1173228 5.18648649,91.4787402 C5.42046332,91.5188976 11.1138996,92.6433071 24.0996139,91.6393701 C34.2776062,90.8362205 40.0490347,90.876378 45.157529,90.9566929 C46.5223938,90.9968504 47.8092664,90.9968504 49.1741313,90.9968504 C51.8258687,90.9968504 50.3830116,90.876378 52.2938224,91.2377953 C53.6976834,91.5188976 54.2046332,92.2015748 60.5220077,93.0448819 C63.0177606,93.4062992 65.2015444,93.5267717 67.1123552,93.5267717 C71.4799228,93.5267717 74.5996139,92.8440945 77.2123552,92.2015748 C78.8111969,91.8401575 80.2150579,91.5188976 81.5799228,91.4787402 C90.7050193,91.1173228 100.688031,91.8401575 100.805019,91.8401575 L100.922008,88.9488189 C100.844015,88.9889764 90.8220077,88.1055118 81.5019305,88.4669291 C79.9030888,88.5472441 78.3822394,88.8685039 76.6274131,89.2700787 C73.0397683,90.0732283 68.5162162,91.0370079 60.9119691,90.0330709 C56.388417,89.3905512 55.5694981,89.0291339 54.5945946,88.7480315 C54.711583,87.2220472 54.984556,83.8086614 55.2185328,79.1503937 C56.5444015,79.511811 57.8312741,79.7125984 59.0401544,79.7125984 C60.872973,79.7125984 62.5888031,79.2708661 64.1096525,78.4677165 C69.8030888,75.415748 71.7528958,67.9866142 71.830888,67.6653543 L72.0648649,66.7417323 L71.1289575,66.7015748 C62.1598456,66.4606299 57.6362934,70.0748031 55.4525097,73.2874016 C55.5694981,69.9543307 55.6084942,66.2598425 55.5694981,62.284252 C55.5694981,61.6417323 55.5694981,59.9952756 55.5694981,58.9913386 C55.8814672,58.9913386 56.1934363,59.0314961 56.4664093,59.0314961 C61.5359073,59.0314961 65.7474903,57.4251969 69.023166,54.3330709 C76.3544402,47.1850394 76.042471,34.615748 76.042471,34.0937008 L76.0034749,33.1299213 L75.1065637,33.3708661 C62.6667954,36.5433071 57.3633205,43.2496063 55.1405405,48.911811 C54.8675676,45.819685 54.43861,43.4905512 53.853668,40.3582677 C53.580695,38.711811 53.2297297,36.8244094 52.8397683,34.4952756 C52.8007722,34.3748031 52.8397683,34.2141732 52.8397683,34.0937008 C55.9984556,33.8527559 58.5722008,32.7283465 60.5220077,30.6 C65.0455598,25.9417323 64.6945946,18.2314961 64.6945946,17.9102362 L64.6555985,16.9464567 L63.7586873,17.2275591 C57.4023166,19.1149606 53.9706564,22.2472441 52.1378378,25.2590551 C51.9818533,14.8984252 53.307722,3.33307087 53.5416988,2.20866142 L53.3857143,0.963779528 L51.5918919,0.361417323 C50.7339768,0.722834646 50.227027,-2.69055118 49.4471042,12.0070866 C49.3301158,14.4968504 48.9401544,18.5125984 48.9011583,20.8015748 C45.1965251,16.5850394 39.269112,15.4606299 38.996139,15.4204724 L38.0602317,15.2598425 L38.0992278,16.223622 C38.1772201,17.9102362 38.4111969,19.396063 38.6841699,20.7614173 C29.7150579,18.392126 20.2,23.411811 19.7710425,23.6527559 L18.9521236,24.0944882 L19.5370656,24.8173228 C27.1023166,34.2141732 34.6675676,36.623622 40.4389961,36.623622 C45.0405405,36.623622 48.5111969,35.0976378 49.9540541,34.2944882 C50.3050193,36.623622 50.6949807,39.5952756 50.9679537,40.9204724 C51.5528958,44.0125984 51.9818533,46.3417323 52.2548263,49.5543307 C51.5528958,48.0685039 50.5779923,46.7433071 49.4081081,45.6188976 C44.1826255,40.719685 36.3833977,41.3220472 36.0714286,41.3622047 L35.1355212,41.4425197 L35.4084942,42.3661417 C39.8930502,56.7425197 48.5501931,58.2283465 51.903861,58.2283465 C52.176834,58.2283465 52.4108108,58.2283465 52.6057915,58.188189 C52.6057915,59.4330709 52.6447876,60.7582677 52.6447876,62.2440945 C52.6447876,64.8141732 52.6057915,67.303937 52.5667954,69.6330709 C51.4749035,67.1433071 49.8760618,65.0149606 47.8092664,63.3283465 C39.8540541,56.9031496 27.8432432,58.9913386 27.3362934,59.0716535 L26.4393822,59.2322835 L26.7903475,60.115748 C33.2637066,76.419685 43.9486486,78.8692913 49.7590734,78.8692913 C50.7339768,78.8692913 51.5918919,78.7889764 52.2548263,78.7086614 C52.0598456,83.0858268 51.7868726,86.3385827 51.6698842,88.0251969 C51.2019305,87.9448819 50.5,87.9047244 49.1741313,87.9047244 C47.8482625,87.9047244 46.56139,87.8645669 45.1965251,87.8645669 C40.0490347,87.784252 34.1606178,87.7440945 23.8656371,88.5472441 C11.3868726,89.5511811 5.81042471,88.4669291 5.65444015,88.4267717 C1.403861,87.9047244 0.740926641,88.5874016 0.272972973,89.0692913 L0.467953668,92.0409449 C0.38996139,92.1212598 1.403861,91.6795276 1.403861,91.6795276 Z M70.1540541,68.1472441 C69.4521236,70.1952756 67.4633205,74.9338583 63.4857143,77.0622047 C61.0289575,78.3874016 58.2212355,78.507874 55.0625483,77.4637795 C55.6864865,75.3354331 58.5722008,68.2677165 70.1540541,68.1472441 Z M74.5216216,35.0574803 C74.4436293,38.1094488 73.6637066,47.5866142 67.9702703,53.1685039 C64.7335907,56.3409449 60.3660232,57.7866142 55.0235521,57.4653543 C54.8285714,54.2929134 55.257529,40.4385827 74.5216216,35.0574803 Z M21.330888,24.576378 C24.0216216,23.3314961 32.6007722,19.9181102 40.0100386,22.8094488 C44.1826255,24.415748 47.3413127,27.8291339 49.369112,32.9291339 C46.7173745,34.4551181 34.3945946,40.0771654 21.330888,24.576378 Z M50.1100386,30.8409449 C47.8872587,26.1425197 44.6895753,22.9700787 40.5169884,21.3637795 C40.4779923,21.3637795 40.4389961,21.323622 40.3610039,21.323622 C40.0490347,20.0787402 39.8150579,18.7133858 39.6980695,17.1472441 C41.6868726,17.7094488 46.4444015,19.3559055 48.823166,23.2511811 C50.1490347,25.419685 50.5779923,27.9496063 50.1100386,30.8409449 Z M37.1633205,42.8480315 C39.3471042,42.8480315 44.6895753,43.2496063 48.3942085,46.7031496 C50.8899614,49.0724409 52.2158301,52.4055118 52.3718147,56.6622047 C50.0320463,56.8629921 41.7648649,56.4212598 37.1633205,42.8480315 Z M51.5528958,32.607874 C51.5528958,32.1661417 51.5528958,31.5637795 51.630888,30.8409449 C51.630888,30.8007874 51.630888,30.7606299 51.630888,30.7204724 C52.0988417,27.507874 54.2436293,22.0062992 63.1737452,19.0346457 C63.0957529,21.2031496 62.5888031,26.3433071 59.4301158,29.596063 C57.519305,31.6440945 54.7895753,32.688189 51.5528958,32.607874 Z M44.1826255,76.580315 C37.5532819,74.6125984 32.1718147,69.0307087 28.5451737,60.4370079 C31.5088803,60.115748 40.7119691,59.5133858 46.9123552,64.5330709 C50.4220077,67.384252 52.4108108,71.6007874 52.8787645,77.1023622 C51.5528958,77.3433071 48.1992278,77.7850394 44.1826255,76.580315 Z" id="Shape"></path>
                    <path d="M23.8266409,96.2976378 C29.2861004,96.2173228 32.6397683,95.9362205 35.642471,95.6952756 C39.8930502,95.3338583 43.2467181,95.0929134 50.6559846,95.5346457 C53.3857143,95.6952756 56.0764479,95.9362205 58.6501931,96.1370079 C63.5247104,96.5385827 68.2822394,96.9401575 72.8837838,96.9401575 C77.2903475,96.9401575 81.5799228,96.5787402 85.7525097,95.5346457 C93.5517375,93.6070866 98.9332046,94.9322835 100.883012,95.5748031 L100.883012,94.7716535 C98.6602317,94.0086614 93.3177606,92.8440945 85.5965251,94.7716535 C77.1733591,96.8598425 68.2042471,96.1370079 58.6891892,95.3740157 C56.0764479,95.1732283 53.3857143,94.9322835 50.6559846,94.7716535 C43.207722,94.3299213 39.8150579,94.6110236 35.5644788,94.9322835 C32.6007722,95.1732283 29.2471042,95.4543307 23.8266409,95.5346457 C15.8714286,95.6149606 9.04710425,94.7314961 4.99150579,94.1692913 C3.08069498,93.9283465 1.52084942,93.7275591 0.350965251,93.6874016 L0.350965251,94.4503937 C1.63783784,94.5307087 3.54864865,94.7716535 4.87451737,94.9322835 C8.96911197,95.4944882 15.8714286,96.4181102 23.8266409,96.2976378 Z" id="Shape"></path>
                    <path d="M85.9864865,97.0606299 C77.5633205,99.1488189 68.5942085,98.4259843 59.0791506,97.6629921 C56.4664093,97.4622047 53.7756757,97.2212598 51.0459459,97.0606299 C43.5976834,96.6188976 40.2050193,96.9 35.9544402,97.2212598 C32.9907336,97.4622047 29.6370656,97.7433071 24.2166023,97.823622 C16.26139,97.9440945 9.43706564,97.0204724 5.38146718,96.4582677 C3.27567568,96.1771654 1.52084942,95.976378 0.38996139,95.976378 L0.38996139,96.7393701 C1.63783784,96.819685 3.82162162,97.0606299 5.3034749,97.2614173 C9.3980695,97.823622 16.3003861,98.7472441 24.2555985,98.6267717 C29.7150579,98.5464567 33.0687259,98.2653543 36.0714286,98.0244094 C40.3220077,97.6629921 43.6756757,97.3818898 51.0849421,97.8637795 C53.8146718,98.0244094 56.5054054,98.2653543 59.0791506,98.4661417 C63.953668,98.8677165 68.7111969,99.2692913 73.3127413,99.2692913 C77.719305,99.2692913 82.0088803,98.907874 86.1814672,97.8637795 C94.253668,95.8559055 99.323166,97.1811024 100.922008,97.7433071 L100.922008,96.9401575 C98.9332046,96.2173228 93.9027027,95.0929134 85.9864865,97.0606299 Z" id="Shape"></path>
                    <path d="M86.7274131,99.7511811 C78.3042471,101.83937 69.3351351,101.116535 59.8200772,100.353543 C57.2073359,100.152756 54.5166023,99.911811 51.7868726,99.7511811 C44.33861,99.3094488 40.9459459,99.5905512 36.6953668,99.911811 C33.7316602,100.152756 30.3779923,100.433858 24.957529,100.514173 C17.0023166,100.634646 10.1779923,99.7110236 6.12239382,99.1488189 C3.47065637,98.7874016 1.403861,98.5866142 0.38996139,98.6669291 L0.38996139,99.3897638 C1.16988417,99.3496063 4.21158301,99.6708661 6.04440154,99.911811 C10.1389961,100.474016 17.0413127,101.397638 24.9965251,101.277165 C30.4559846,101.19685 33.8096525,100.915748 36.8123552,100.674803 C41.0629344,100.313386 44.4166023,100.032283 51.8258687,100.514173 C54.5555985,100.674803 57.246332,100.915748 59.8200772,101.116535 C64.6945946,101.51811 69.4521236,101.919685 74.053668,101.919685 C78.4602317,101.919685 82.7498069,101.558268 86.9223938,100.514173 C94.1366795,98.7070866 98.8942085,99.7110236 100.922008,100.393701 L100.922008,99.5905512 C98.6212355,98.8275591 93.8637066,97.984252 86.7274131,99.7511811 Z" id="Shape"></path>
                </g>
            </g>
        </g>
    </g>
</svg>