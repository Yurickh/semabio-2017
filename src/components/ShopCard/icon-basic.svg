<?xml version="1.0" encoding="UTF-8"?>
<svg width="91px" height="70px" viewBox="0 0 91 70" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 45.2 (43514) - http://www.bohemiancoding.com/sketch -->
    <title>icon-basic</title>
    <desc>Created with Sketch.</desc>
    <defs></defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="BetterDesk" transform="translate(-178.000000, -3318.000000)">
            <g id="icon-basic" transform="translate(178.000000, 3318.000000)">
                <polygon id="Shape" fill="#C3AFB1" points="43.9364741 53.5082614 25.6296099 69.8312722 26.0029447 40.6161481"></polygon>
                <polygon id="Shape" fill="#F4F4F4" points="0 31.2934992 19.2557562 40.9993075 29.0137253 45.9176839 64.6542636 63.4181961 90.0697719 0"></polygon>
                <polygon id="Shape" fill="#D6D5D5" points="90.0697719 0 19.0391388 40.9291095 25.6804 69.8312722 28.8269571 45.8390648"></polygon>
            </g>
        </g>
    </g>
</svg>