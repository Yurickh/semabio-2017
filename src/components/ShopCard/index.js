import React, { Component } from 'react'
import PropTypes from 'prop-types'

import Price from '../Price'
import Button from '../Button'

import iconOne from './icon-1.svg'
import iconTwo from './icon-2.svg'
import iconThree from './icon-3.svg'
import iconFour from './icon-4.svg'
import iconFive from './icon-5.svg'

import './styles.css'

const Levels = [
  {
    variant: '-red',
    image: iconOne,
  }, {
    variant: '-orange',
    image: iconTwo,
  }, {
    variant: '-green',
    image: iconThree,
  }, {
    variant: '-blue',
    image: iconFour,
  }, {
    variant: '-purple',
    image: iconFive,
  }
]

class ShopCard extends Component {
  render() {
    const level = Levels[this.props.level]
    return (
      <div className="shop-card">
        <img className="icon"
          alt=""
          src={level.image}
        />
        <h3 className={`title ${level.variant}`}>
          {this.props.title}
        </h3>

        <div className="price">
          <Price>{this.props.price}</Price>
        </div>

        <div className="description">
          {this.props.description.map(phrase => <div key={phrase}>{phrase}</div>)}
        </div>

        {/*<Button color="green"> Comprar </Button>*/}
      </div>
    )
  }
}

ShopCard.propTypes = {
  title: PropTypes.string.isRequired,
  price: PropTypes.number.isRequired,
  level: PropTypes.oneOf([0, 1, 2, 3, 4]).isRequired,
  description: PropTypes.arrayOf(PropTypes.string),
}

ShopCard.defaultProps = {
  description: []
}

export default ShopCard
