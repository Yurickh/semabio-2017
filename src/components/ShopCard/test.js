import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import ShopCard from '../ShopCard'

describe('ShopCard', () => {
  it('should render correctly', () => {
    const button = shallow(
      <ShopCard
        title="test title"
        price={ 3.14 }
        level={2}
      />
    )
    expect(toJSON(button)).toMatchSnapshot()
  })
})
