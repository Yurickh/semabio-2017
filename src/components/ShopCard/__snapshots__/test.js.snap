// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShopCard should render correctly 1`] = `
<div
  className="shop-card"
>
  <img
    alt=""
    className="icon"
    src="icon-3.svg"
  />
  <h3
    className="title -green"
  >
    Pacote 
  </h3>
  <div
    className="price"
  >
    <Price>
      3.14
    </Price>
  </div>
  <div
    className="description"
  />
  <Link
    replace={false}
    to="undefinedcomprar/undefined"
  >
    <Button
      color="green"
      onClick={[Function]}
    >
       Comprar 
    </Button>
  </Link>
</div>
`;
